src/Test/config.jl

```diff
@@ -1,6 +1,6 @@
 struct TestConfig{T <: Number}
-    atol::Float64 # absolute tolerance for ...
-    rtol::Float64 # relative tolerance for ...
+    atol::Real # absolute tolerance for ...
```

Comment:
Why not `atol::T` ?

Reply:
Here, the type of the field is not concrete

Reply:
Can I leave it as ::Real in the constructor function and use ::T just for the field?

Reply:
IIRC it will automatically convert when setting the field in construction

Reply:
Yes, that sounds good, what's weird is that we allow `T` to be `Number` but it does not makes sense to have a complex tolerance

Reply:
do we ever want any complex TestConfigs? could we change Number to Real?